setup.py

```diff
@@ -1,31 +1,33 @@
-import os
+""" Python setup configuration file for Open-SIR"""
 from setuptools import setup
 
-# Utility function to read the README file.
-# Used for the long_description.  It's nice, because now 1) we have a top level
-# README file and 2) it's easier to type in the README file than to put a raw
-# string in below ...
-def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
-
-
 setup(
     name="opensir",
     version="1.0.0",
     author="Open-SIR community",
     author_email="open.sir.project@gmail.com",
     description=(
         "Open-SIR is an Open Source Python project for modelling"
-        "pandemics and infectious diseases using Compartmental"
-        "Models"
+        " pandemics and infectious diseases using Compartmental"
```

Comment:
I think the description here should be shorter here. E.g "Open Source pandemics modeling using Compartmental Models".
Also, there's a typo:
`sed s/modelling/modeling/`